samples/k8s-multi-version.md

```diff
@@ -0,0 +1,178 @@
+## Seldon V2 Kubernetes Multi Version Artifact Examples
+
+We have a Triton model that has two version folders
+
+Model 1 adds 10 to input, Model 2 multiples by 10 the input. The structure of the artifact repo is shown below:
+
+```
+config.pbtxt
+1/model.py <add 10>
+2/model.py <mul 10>
+
+```
+
+```python
+import os
+os.environ["NAMESPACE"] = "seldon-mesh"
+```
+
+```python
+MESH_IP=!kubectl get svc seldon-mesh -n ${NAMESPACE} -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+MESH_IP=MESH_IP[0]
+import os
+os.environ['MESH_IP'] = MESH_IP
+MESH_IP
+```
+
+```
+'172.19.255.1'
+
+```
+
+### Model
+
+```bash
+cat ./models/multi-version-1.yaml
+```
+
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Model
+metadata:
+  name: math
+spec:
+  storageUri: "gs://seldon-models/scv2/samples/triton_22-11/multi-version"
+  artifactVersion: 1
+  requirements:
+  - triton
+  - python
+
+```
+
+```bash
+kubectl create -f ./models/multi-version-1.yaml -n ${NAMESPACE}
```

Comment:
:thought_balloon: Probably worth using `apply` here as we expect to update this later; I realise this is generated from the notebook